index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script
+      src="https://kit.fontawesome.com/a70cf7601d.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="./css/styles.css" />
+    <title>Personal Portfolio</title>
+  </head>
+
+  <body>
+    <section>
+      <header class="header">
+        <nav class="nav">
+          <a href="">Welcome</a>
+          <i class="fas fa-bars"></i>
+        </nav>
+      </header>
+      <div class="hero-heading">
+        <h1>Hey there, I am Dennis.</h1>
+        <h1 id="title">I'm a software Developer</h1>
+        <p>
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+        <div class="social-icons">
+          <a href="#"><img src="/images/icons/gitHub.png" alt="github" /></a>
+          <a href="#"><img src="/images/icons/linkedin.png" alt="linkdln" /></a>
+          <a href="#"
+            ><img src="/images/icons/angelist.png" alt="engelist"
+          /></a>
+          <a href="#"><img src="/images/icons/twitter.png" alt="twitter" /></a>
+          <a href="#"><img src="/images/icons/medium.png" alt="medium" /></a>
```

Comment:
- [ ] Good work on using all the required icons 👍. However your icons are currently not displaying when viewed on the browser. I suspect this is because you need to use relative links for your image tags. 
something like this for the a tags would be better:
 ```suggestion
          <a href="#"><img src="./images/icons/gitHub.png" alt="github" /></a>
          <a href="#"><img src="./images/icons/linkedin.png" alt="linkdln" /></a>
          <a href="#"
            ><img src="./images/icons/angelist.png" alt="engelist"
          /></a>
          <a href="#"><img src="./images/icons/twitter.png" alt="twitter" /></a>
          <a href="#"><img src="./images/icons/medium.png" alt="medium" /></a>
```
_You can read more about relative and absolute paths in this 👉 [link](https://www.codegrepper.com/code-examples/html/html+relative+path)_

Reply:
This has been resolved. Thank you